src/traits/json.rs

```diff
@@ -106,3 +106,20 @@ impl ToJson for BlockchainVarV1 {
         }))
     }
 }
+
+impl ToJson for BlockchainTxnTransferHotspotV1 {
+    fn to_json(&self) -> Result<serde_json::Value> {
+        let seller = PubKeyBin::from_vec(self.seller.as_slice()).to_string();
```

Comment:
why use this? there's the to_b58 functions that we use to display gateway, seller, and buyer keys and the values in the txn are already pubkey_bins.. something like `self.seller.to_b58()?`

Reply:
If I use `to_b58()` it doesn't prefix the appropriate bytes and it displays wrong. 

It's in my example output above (and I fixed it after), but basically if I change it to:
```
        let seller = self.seller.to_b58()?;
        let gateway = self.gateway.to_b58()?;
        let buyer = self.buyer.to_b58()?;
```

This happens:
```
./helium-wallet hotspots transfer sell 112AMbwEAp4QyZeBQYuTMt8wa5W6ceK1xNjG59duxqB6Dx7fS1c4 13U1qigMC832L2oJLYqEYEdBH1JBMNqbRYZ6RuduNr6ntsKP7om`
Object({
    "amount_to_seller": Number(
        0,
    ),
    "buyer": String(
        "3U1qigMC832L2oJLYqEYEdBH1JBMNqbRYZ6RuduNr6ntuw7G8g",
    ),
    "buyer_nonce": Number(
        3,
    ),
    "fee": Number(
        55000,
    ),
    "gateway": String(
        "12AMbwEAp4QyZeBQYuTMt8wa5W6ceK1xNjG59duxqB6Dx3CGWe9",
    ),
    "seller": String(
        "33yVfiCKZKTxHgWY6UQ8uD6CX2j9q5e2BNjZGCdmcUMLPPir3e",
    ),
})
```

The current way, I get:
```
helium-wallet  hotspots transfer sell 112AMbwEAp4QyZeBQYuTMt8wa5W6ceK1xNjG59duxqB6Dx7fS1c4 13U1qigMC832L2oJLYqEYEdBH1JBMNqbRYZ6RuduNr6ntsKP7om`
Object({
    "amount_to_seller": Number(
        0,
    ),
    "buyer": String(
        "13U1qigMC832L2oJLYqEYEdBH1JBMNqbRYZ6RuduNr6ntsKP7om",
    ),
    "buyer_nonce": Number(
        3,
    ),
    "fee": Number(
        55000,
    ),
    "gateway": String(
        "112AMbwEAp4QyZeBQYuTMt8wa5W6ceK1xNjG59duxqB6Dx7fS1c4",
    ),
    "seller": String(
        "133yVfiCKZKTxHgWY6UQ8uD6CX2j9q5e2BNjZGCdmcUMLSMubn5",
    ),
})
```

Reply:
I meant more like `PubKeyBin::from_vec(data).to_b58()` perhaps? Making it clearer what you're getting instead of a slice to_string

Reply:
Yes, you're right. Fixed that one.